usaspending_api/etl/management/commands/load_historical.py

```diff
@@ -22,27 +33,43 @@ class Command(load_base.Command):
     def add_arguments(self, parser):
 
         super(Command, self).add_arguments(parser)
-        parser.add_argument('--action_date_begin', type=to_date, default=None, help='First action_date to get - YYYY-MM-DD')
-        parser.add_argument('--action_date_end', type=to_date, default=None, help='Last action_date to get - YYYY-MM-DD')
-        parser.add_argument('--cgac', default=None, help='awarding toptier agency code')
+        parser.add_argument('--contracts', action='store_true', help='Load contracts (not FA')
+        parser.add_argument('--financial_assistance', action='store_true', help='Load financial assistance (not contracts)')
+        parser.add_argument('--action_date_begin', type=parse_date, default=None, help='First action_date to get - YYYY-MM-DD')
+        parser.add_argument('--action_date_end', type=parse_date, default=None, help='Last action_date to get - YYYY-MM-DD')
+        parser.add_argument('--cgac', default=None)
 
     def handle_loading(self, db_cursor, *args, **options):
 
         submission_attributes = SubmissionAttributes()
         submission_attributes.usaspending_update = datetime.now()
         submission_attributes.save()
 
-        # File D1
-        sql = 'SELECT * FROM detached_award_procurement WHERE true'
+        if options['contracts'] and options['financial_assistance']:
+            raise CommandError('Default is to load both contracts and financial_assistance')
+
+        if not options['financial_assistance']:
+            procurement_data = self.broker_data(db_cursor, 'detached_award_procurement', options)
+            load_base.load_file_d1(submission_attributes, procurement_data, db_cursor)
+
+        if not options['contracts']:
```

Comment:
That's the idea - if you don't specify `--contracts` or `--financial_assistance`, you get both.

Alternately, it could load nothing if you don't specify either.  I could go either way...